keyboards/bbrfkr/dynamis/dynamis.c

```diff
@@ -16,18 +16,9 @@
 
 #include "dynamis.h"
 
-#ifdef ENCODER_ENABLE
+#ifndef ENCODER_ENABLE
 bool encoder_update_kb(uint8_t index, bool clockwise) {
-    if (!encoder_update_user(index, clockwise)) {
-        return false; 
-    }
-   if (index == 0) { /* First encoder */
-      if (clockwise) {
-         tap_code(KC_WH_D);
-      } else {
-         tap_code(KC_WH_U);
-      }
-   }
-   return false;
+    if (!encoder_update_user(index, clockwise)) { return false; }
+    return true;
```

Comment:
This this has a default behavior:
```suggestion
      if (clockwise) {
         tap_code(KC_WH_D);
      } else {
         tap_code(KC_WH_U);
      }
   }
   return true;
```

Reply:
Thank you. I added the default behavior of encoder.